src/edfi-performance-test/edfi_performance_test/tasks/pipeclean/descriptors.py

```diff
@@ -16,10 +16,8 @@ class DescriptorPipecleanTestBase(EdFiPipecleanTestBase):
     def __init__(self, descriptor: str, parent, *args, **kwargs):
         super(DescriptorPipecleanTestBase, self).__init__(parent, *args, **kwargs)
 
-        self.update_attribute_name = "codeValue"
```

Comment:
@JBrenesSimpat can't we simply handle this by updating description field
self.update_attribute_name = "description"
self.update_attribute_value = random_chars(15)